wordpress-mobile/WordPress-Android
# Open keyboard on expand click on Full screen comment

Fixes #10235

This PR adds functionality to open the keyboard when the user clicks on the "Expand" button on the comment detail. The Full screen dialog now automatically opens the keyboard. I've extracted the logic into a trivial view model with a simple test.

To test:
* Go to MySite/Comments
* Click on a comment
* Notice that the keyboard is closed
* Click on the "Expand" button
* Notice that the keyboard is now opened

Update release notes:
